### ISSUE TITLE
[mapdb] Add maintenance commands to delete entries

### ISSUE THREAD

**Issue by dilyanpalauzov** [openhab/openhab-addons#15102]
When a device is permanently removed from openHAB I want to remove its records from MapDB.

Please provide maintenance commands to:
* list all keys in mapdb
* list all keys and values in mapdb
* delete a key in mapdb

**Comment by J-N-K** [comment]
I don't think this should be the way to go. If possible, a database service should implement ModifiablePersistenceService and allow removing data from the REST API. That way all services behave the same way and we can have a unified UI for that.

**Comment by jlaur** [comment]
I agree that a unified way would be superior for maintenance. Just as input, there are already some console commands for JDBC persistence: https://www.openhab.org/addons/persistence/jdbc/#maintenance
